pkg/agent/core/ngt/handler/grpc/handler.go

```diff
@@ -1175,7 +1175,7 @@ func (s *server) IndexInfo(ctx context.Context, _ *payload.Empty) (res *payload.
 	}()
 	return &payload.Info_Index_Count{
 		Stored:      uint32(s.ngt.Len()),
-		Uncommitted: uint32(s.ngt.InsertVQueueLen()),
+		Uncommitted: uint32(s.ngt.InsertVQueueBufferLen()),
```

Comment:
Doesn't need to add `InsertVQueueChannelLen`?

Reply:
In this case, `InsertVQueueChannelLen` data is not ready for create index, so we can ignore it at this time, but we should add `DeleteVQueueBufferLen` thank you for mention it.

Reply:
I see.